pixmappy/PixelMapCollection.py

```diff
@@ -374,33 +411,49 @@ def __init__(self, name, pmap, projection, scale=1.):
         self.projection = projection
         self.scale = scale
         self.dest = None
-        return
+
     def reprojectTo(self, projection):
         self.dest = projection
-        return
-    def toSky(self, xy, c=None):
+
+    def toSky(self, x, y, c=None):
         ''' Return sky coordinates corresponding to array
         '''
-        return self.projection.toSky(self.pmap(xy,c) * self.scale)
-    def toPix(self, coords, c=None, guess=np.array([0.,0.])):
-        ''' Return pixel coordinates corresponding to input SkyCoords.
+        # This is the front end user interface.  Let's not modify the user's input x,y values.
```

Comment:
Good point.  I think `inverse` and `jacobian` are already safe (or at least jacobian would be if `__call__` were safe).  But you're right about `__call__`.  I just [fixed](7a1f322) that to do the same copy that we do for `toSky`.